counterpartylib/lib/transaction.py

```diff
@@ -449,7 +449,13 @@ def construct (db, tx_info, encoding='auto',
             data_output_size = data_output_size + len(data)
     else:
         data_output_size = p2pkhsize   # Pay‐to‐PubKeyHash (25 for the data?)
-    outputs_size = (p2pkhsize * len(destination_outputs)) + (len(data_array) * data_output_size)
+
+    # destination + data + 'change'
+    # Strictly we can't determine if the change output is contained in this transaction.
+    # But in almost all cases, it will be contained.
+    outputs_size = (p2pkhsize * len(destination_outputs)) + (len(data_array) * data_output_size) + p2pkhsize
```

Comment:
This will lead to overestimation for non-change sends.... these could happen a lot on really fragmented wallets that have a lot of small outputs.... although this could lead to better estimation on a common 280 bytes op (i.e. a standard enhanced send) small outputs would become unusable.

Gotta weigh which case is the most common and approve or reject based on that.